### ISSUE TITLE
Enabling "Randomize port on launch" may impact download speeds in certain cases

### ISSUE THREAD

**Issue by Coding4Coffee** [transmission/transmission#4942]
### What is the issue?

Whilst it sounds attractive from an anti-tracking aspect to randomize the port you use on each launch; this could lead to slower download speeds.

This being due to DHT, peers, and trackers keeping your old port in their lists. They will continue trying to send data to your old port until that peer has announced to the tracker to get your new IP:Port. This could be 30 minutes or longer. Whilst it wouldn't be a big issue with a large swarm, it would impact torrents with a smaller swarm, and might have users complaining about slow speeds.

I think this caveat should be stated in the docs, but the average user doesn't read docs, (All those guilty, say I 😉) and may enable it regardless. My suggestion is to have a small note in the same style as the "Automatically map port" setting, pointing out that caveat below the option in the Network GUI.

"May impact download speeds in certain cases"

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.0-dev (671f835228)

**Comment by ckerr** [comment]
If anyone wants to work on this --

* I'd take a docs/ PR for this for 4.0.x

* I'd take a one-liner in prefs/settings for 4.1.x (semver minor and would need to go to translators)

**Comment by Coeur** [comment]
Let's go for the settings note: I wasn't aware of that fact, so it should have direct visibility.

**Comment by ckerr** [comment]
@Coeur 4.1.0 freeze is coming soon, looks like you milestoned this for that release. Do you have bandwidth to make a PR?

**Comment by Coeur** [comment]
No, I'm still minimising my participation until that code_style.sh is addressed one way or another (like #7573).
